saddle-core/src/main/scala/org/saddle/array/package.scala

```diff
@@ -47,20 +47,18 @@ package object array {
     *
     * @param from
     *   start index, inclusive
-    * @param until
+    * @param to
```

Comment:
why change here?
In ranges in the scala std lib, `1 to 10` or `1 until 10`, `to` is inclusive and `until` is exclusive.
I would rather stick to that convention.